modules/test/playwright/pages/commerce/commerce-tax-engine-web/TaxCategoriesPage.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {liferayConfig} from '../../../liferay.config';
+import {searchTableRowByValue} from '../../users-admin-web/OrganizationUsersPage';
+
+export class TaxCategoriesPage {
+	readonly deleteMenuItem: Locator;
+	readonly newButton: Locator;
+	readonly page: Page;
+	readonly taxCategoriesTable: Locator;
+	readonly taxCategoriesTableRow: (
+		colPosition: number,
+		value: string,
+		strictEqual?: boolean
+	) => Promise<{column: Locator; row: Locator}>;
+	readonly taxCategoriesTableRowActions: (
+		screenName: string
+	) => Promise<Locator>;
+	readonly taxCategoriesTableRowLink: (
+		screenName: string
+	) => Promise<Locator>;
+
+	constructor(page: Page) {
+		this.deleteMenuItem = page.getByRole('link', {
+			name: 'Delete',
+		});
+		this.newButton = page.getByRole('link', {name: 'Add Tax Category'});
+		this.page = page;
+		this.taxCategoriesTable = page.locator(
+			'#_com_liferay_commerce_product_tax_category_web_internal_portlet_CPTaxCategoryPortlet_cpTaxCategoriesSearchContainer'
+		);
+		this.taxCategoriesTableRow = async (
+			colPosition: number,
+			value: string,
+			strictEqual: boolean = false
+		) => {
+			return await searchTableRowByValue(
+				this.taxCategoriesTable,
+				colPosition,
+				value,
+				strictEqual
+			);
+		};
+		this.taxCategoriesTableRowActions = async (name: string) => {
+			const usersTableRow = await this.taxCategoriesTableRow(1, name);
+
+			if (usersTableRow && usersTableRow.column) {
+				return usersTableRow.row.getByRole('button');
+			}
+
+			throw new Error(`Cannot locate tax category row with name ${name}`);
+		};
+		this.taxCategoriesTableRowLink = async (name: string) => {
+			const usersTableRow = await this.taxCategoriesTableRow(1, name);
+
+			if (usersTableRow && usersTableRow.column) {
+				return usersTableRow.column.getByRole('link', {
+					name,
+				});
+			}
+
+			throw new Error(`Cannot locate tax category with name ${name}`);
+		};
+	}
+
+	async goto() {
+		const portletId =
+			'com_liferay_commerce_product_tax_category_web_internal_portlet_CPTaxCategoryPortlet';
+
+		const params = new URLSearchParams();
+
+		params.append('p_p_id', portletId);
+
```

Comment:
can you move this in the `portletUrls.ts` file? and if you look to `systemSettings` you can create the string already with the params